common/core/gpudevice.cpp

```diff
@@ -201,6 +201,8 @@ void GpuDevice::DisplayManager::HotPlugEventHandler() {
           "UpdateDisplayState.");
       UpdateDisplayState();
     }
+    /* Adding sleep to avoid busy looping before a retry */
+    usleep(1000);
```

Comment:
Tapani, Its not a single system call, HotPlugEventHandler gets called multiple times (even without actual hotplug, I didn't understand that behaviour!!), we have observed this when we put prints there. If lseek can get rid of loop itself we can get rid of that sleep at cost of 'one more sys call at runtime' (its not a single call).